index.html

```diff
@@ -13,6 +13,7 @@
     <link rel="manifest" id="manifest-placeholder">
 
     <script>
+        window.EXCALIDRAW_ASSET_PATH = 'libs/';
```

Comment:
Does it add the excalidraw-assets directory when composing the path?

Reply:
[Yup, needs to be here ideally](https://github.com/jitsi/excalidraw/blob/master/src/packages/excalidraw/README.md#installation:~:text=If%20you%20want%20to%20load%20assets%20from%20a%20different%20path%20you%20can%20set%20a%20variable%20window.EXCALIDRAW_ASSET_PATH%20depending%20on%20environment%20(for%20example%20if%20you%20have%20different%20URL%27s%20for%20dev%20and%20prod)%20to%20the%20url%20from%20where%20you%20want%20to%20load%20the%20assets.)